sass/main.scss

```
@@ -71,3 +71,36 @@ img {
     margin-bottom: 0.5rem;
   }
 }
+
+table {
+  border-collapse: collapse;
+  margin: 25px auto;
+  font-size: 0.9em;
+  font-family: sans-serif;
+  min-width: 400px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.15);
+}
+
+thead tr {
+  background-color: var(--accent-color);
+  color: #ffffff;
+  text-align: left;
+}
+
+table th,
+table td {
+    padding: 10px 12px;
+}
+
+
+table tbody tr {
+    border-bottom: 1px solid #dddddd;
+}
+
+table tbody tr:nth-of-type(even) {
+    background-color: #f3f3f3;
+}
+
+table tbody tr:last-of-type {
+    border-bottom: 2px solid #009879;
```

This is pretty much the same as the theme colour right?

```suggestion
    border-bottom: 2px solid var(--accent-color);
```